pkg/plugins/builtin/email/email.go

```diff
@@ -0,0 +1,107 @@
+package email
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/smtp"
+	"strings"
+	"text/template"
+
+	"github.com/ovh/utask/pkg/plugins/taskplugin"
+)
+
+// the email plugin send email
+var (
+	Plugin = taskplugin.New("email", "0.1", exec,
+		taskplugin.WithConfig(validConfig, Config{}),
+	)
+)
+
+// Config is the configuration needed to send an email
+type Config struct {
+	SMTPUsername string   `json:"smtp_username"`
+	SMTPPassword string   `json:"smtp_password"`
+	SMTPPort     uint     `json:"smtp_port,omitempty"`
```

Comment:
```suggestion
	SMTPPort     uint16     `json:"smtp_port,omitempty"`
```

Reply:
Why don't use default system based builtin value?

Reply:
Because an TCP destination port is officially a 16 bit unsigned integer

https://tools.ietf.org/html/rfc793#page-15

Reply:
And then, the unmarshal will fail if someone provides 99999 as TCP port instead of failing on smtp.SendMail